src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.tsx

```diff
@@ -77,11 +81,11 @@ function BaseAutoCompleteSuggestions<TSuggestion>(
         [isLargeScreenWidth, suggestions.length, windowWidth],
     );
     useEffect(() => {
-        rowHeight.value = withTiming(measureHeightOfSuggestionRows(suggestions.length, isSuggestionPickerLarge), {
+        rowHeight.value = withTiming(measureHeightOfSuggestionRows(suggestions.length + (isSearchingForMentions ? 1 : 0), isSuggestionPickerLarge), {
```

Comment:
Can we move row number calculations to separate line? It would be more clear in my opinion
```suggestion
        // add 1 to the expected row number if we are searching for mentions - this will add place for loading spinner at the bottom of the list
        const expectedRowNumber = suggestions.length + (isSearchingForMentions ? 1 : 0);
        rowHeight.value = withTiming(measureHeightOfSuggestionRows(expectedRowNumber, isSuggestionPickerLarge), {
```